server/src/routers/conversation.router.ts

```diff
@@ -41,6 +46,43 @@ conversationRouter.get(
   }),
 );
 
+conversationRouter.get(
+  '/:id/all',
+  authMiddleware,
+  requestHandlerMiddleware(async (req, res) => {
+    const currentUser = getUserFromRequestContext(req);
+    const endpointId = req.params.id;
+    if (currentUser.id !== endpointId) {
```

Comment:
Not needed, pass the user id to the function that fetches the conversation and use it there

Reply:
You have already done that